### PR TITLE
Changes for deactivate account

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/5937

*Description of changes:*
- Changes to support deactivate account

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/src/auth_api/services/affiliation.py [hunk 1]
```diff
@@ -113,9 +106,21 @@ def find_affiliated_entities_by_org_id(org_id):
...
+        current_app.logger.debug('>find_visible_affiliations_by_org_id')
         return filtered_affiliations
 
+    @staticmethod
+    def find_affiliations_by_org_id(org_id):
+        """Return business affiliations for the org."""
+        data = []
+        affiliation_models = AffiliationModel.find_affiliations_by_org_id(org_id)
+        if affiliation_models is None:
+            raise BusinessException(Error.DATA_NOT_FOUND, None)
```
Comment (saravanpa-aot) [inline auth-api/src/auth_api/services/affiliation.py:118]: is it intentional to raise exception? instead of empty list?

**Review by saravanpa-aot — APPROVED** [review]

File: auth-api/src/auth_api/services/affiliation.py (hunk 1 above)
Comment (sumesh-aot) [reply auth-api/src/auth_api/services/affiliation.py:118]: Good catch, will remove it

**Review by karthik-aot — APPROVED** [review]

File: auth-api/migrations/versions/3b485a23d057_deactivate_permissions.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def upgrade():
+    permissions_table = table('permissions',
+                              column('membership_type_code', sa.String(length=15)),
+                              column('org_status_code', sa.String(length=25)),
+                              column('actions', sa.String(length=100)))
+    op.execute('delete from permissions where actions=\'deactivate_account\'')
+    op.execute('select setval(\'permissions_id_seq\', 78, true);')  # No need to include this next time onwards
+    op.bulk_insert(
```
Comment (sumesh-aot) [inline auth-api/migrations/versions/3b485a23d057_deactivate_permissions.py:26]: next time onwards just insert as usual withput any id

**Review by saravanpa-aot — APPROVED** [review]

File: auth-api/src/auth_api/services/affiliation.py (hunk 1 above)
Comment (sumesh-aot) [reply auth-api/src/auth_api/services/affiliation.py]: Fixed in new PR

**Pull request merged**